public/createtopic.php

```diff
@@ -63,7 +64,7 @@ class="w-full"
     <?php
     echo "</td></tr>";
 
-    $loadingIconSrc = asset('assets/images/icon/loading.gif');
+    $loadingIcon = Blade::render('<x-fas-spinner id="preview-loading-icon" class="animate-spin opacity-0 transition-all duration-200" alt="Loading..." />');
```

Comment:
`alt` appears to not be a valid attribute for SVG elements.
Based on what I'm reading, it seems like the best course of action here may be to remove the `alt` attribute and hardcode an `aria-hidden="true"` to the element. This should work because these loading spinners are almost always short-lived. We should probably apply this to all the new loading spinners.

Reply:
Makes sense!